### PR TITLE
server: move proxystore up one level

### PR DESCRIPTION
**Pull request by mcluseau**
@astoycos move proxystore up one level so it's more obvious that it's the core thing of kpng's server side.

### CODE REVIEW

**Comment by astoycos** [comment]
Sweet I think this is nice for readability :)

File: server/jobs/api2localdiff/api2localdiff.go [hunk 1]
```diff
@@ -1,17 +0,0 @@
-/*
```
Comment (astoycos) [inline server/jobs/api2localdiff/api2localdiff.go:1]: Why Was this here in the first place? It seems like a place to implement API -> Local Streaming AND diffing?

**Comment by astoycos** [comment]
/lgtm 

Thanks for this bit!

File: server/jobs/api2localdiff/api2localdiff.go (hunk 1 above)
Comment (mcluseau) [reply server/jobs/api2localdiff/api2localdiff.go:1]: an old (1y+) mistake I think, never had any kind of code
